### PR TITLE
Feature/support only mendix datatypes

### PR DESCRIPTION
**Pull request by roland-mendix**

### CODE REVIEW

File: javasource/databaseconnector/impl/ColumnInfo.java [hunk 1]
```diff
@@ -1,12 +1,16 @@
 package databaseconnector.impl;
 
+import com.mendix.systemwideinterfaces.core.meta.IMetaPrimitive.PrimitiveType;
+
 public class ColumnInfo {
   private int index;
```
Comment (nerush) [inline javasource/databaseconnector/impl/ColumnInfo.java]: Could you please these fields final ?

File: javasource/databaseconnector/impl/ResultSetReader.java [hunk 2]
```diff
@@ -40,8 +48,44 @@ public ResultSetReader(final ResultSet resultSet) {
...
+  	  switch (columnInfo.getType()) {
+  	    case Integer:
+  	      columnValue = rs.getInt(index);
+  	      break;
+  	    case AutoNumber:
+  	    case Long:
+    		  columnValue = rs.getLong(index);
+    		  break;
+  	    case DateTime:
+          Timestamp timeStamp = rs.getTimestamp(index, calendar);
```
Comment (nerush) [inline javasource/databaseconnector/impl/ResultSetReader.java]: These lines are not formatted.

File: javasource/databaseconnector/impl/ResultSetReader.java [hunk 3]
```diff
@@ -40,8 +48,44 @@ public ResultSetReader(final ResultSet resultSet) {
...
+  	    case Currency:
+    		  columnValue = rs.getDouble(index);
+    		  break;
+  	    case HashString:
+  	    case Enum:
+  	    case String:
+    		  columnValue = rs.getString(index);
+    		  break;
+  	    case Binary:
+          try (InputStream inputStream = rs.getBinaryStream(index)) {
```
Comment (nerush) [inline javasource/databaseconnector/impl/ResultSetReader.java]: These lines are also not formatted.

File: javasource/databaseconnector/actions/ExecuteQueryToJson.java [hunk 4]
```diff
@@ -1,91 +0,0 @@
-// This file was generated by Mendix Modeler.
```
Comment (nerush) [inline javasource/databaseconnector/actions/ExecuteQueryToJson.java]: This class should be removed from UserActionsRegistrar

**Comment by roland-mendix** [comment]
[820464] Make Database Connector with the simple "Execute query" (to entity) and "Execute statement" actions ready to release publicly.

**Pull request merged**